src/_content/posts/introducing-keynote-mario-munoz.md

```diff
@@ -0,0 +1,59 @@
+---
+author: DjangoCon US Communications Team
+category: General
+date: 2024-08-01 06:00:00
+cover:
+    url: /assets/img/blog/mario_munoz.jpg
+    alt: "Mario Munoz"
+layout: post
+title: "Announcing Our Keynotes: Mario Munoz"
+---
+
+### We're honored to announce our seecond keynote speaker for DjangoCon US 2024: Mario Munoz!
+
+We had the opportunity to sit down with Mario and ask her a few questions about his work, his upcoming talk, and what shehee's looking forward to at DjangoCon US 2024.
+
+## Can you tell us how you use Django
+
+>When preparing my first talk on "why I didn't choose Django," I thought I should at least build a Django app before getting into it... As a result, I built an app to hold my presentation slides (also using reveal.js), and ever since, I've continued using it for the past two years.
+
+## Can you give us a teaser about what your talk is about?
+
+>I am very interested in the intersection of people and technology. What role does community have in our technological journey? How do we see ourselves within the context of a "technical" community?  At a recent conference (North Bay Python), I spoke about building bridges between communities. My talk at DjangoCon will be an extension of this. My current working title is "How to be a developer, and other lies we tell ourselves." We'll be going through a journey that brings us in and out of different mindsets as developers, and what role community plays in all of this. As many of my talks, I often have many biographical/personal/meta examples...
+
+## What are you most looking forward to at DjangoCon US 2024?
+
+>I am most excited about engaging with the community. I am currently involved in the Djangonaut program and look forward to seeing fellow Djangonauts there. I've also seen that the talks have been posted, and some of them look fantastic. I'm interested in all things hypermedia, so definitely will be catching those htmx-related talks.
+
+## What are your favorite topics at conferences?
+
+>As mentioned, I'm excited about the hypermedia. But I always like hearing about broad topics, such as the past, present, and future of Django talk by Will, or Jacob's talk on what the DSF could accomplish with 4x the budget.
```

Comment:
It would be useful to link to those talks

Reply:
I didn't link to them immediately because I didn't want to hard-code them and I didn't know the e11nty way to do that yet.

Reply:
asked about it in slack